addons/udes_scanned_by_tracking/models/stock_move_line.py

```diff
@@ -0,0 +1,56 @@
+# # -*- coding: utf-8 -*-
+
+from odoo import api, models, fields, tools, _
+from datetime import datetime
+
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class StockMoveLine(models.Model):
+    _inherit = "stock.move.line"
+
+    u_done_by = fields.Many2one(
+        "res.users", "Scanned by", help="ID of the user to complete the pack op.", index=True
+    )
+
+    u_done_datetime = fields.Datetime(
+        "Completion datetime", help="Date and time the operation was completed.", index=True
+    )
+
+    @staticmethod
+    def _now_for_tracking_data():
+        """Can be overwritten to change the tracking resolution"""
+        return datetime.now()
+
+    @api.model
+    def _add_user_tracking_data(self, vals):
+        """Inject/overwrite user tracking data into vals"""
+
+        if vals.get("qty_done", 0) > 0:
+            if not self.env.real_uid:
```

Comment:
Worth adding a comment that this field is expected to be added in Odoo core.

Reply:
Fixed